gcn/notices/einstein_probe/wxt/alert.schema.example.json

```diff
@@ -6,6 +6,9 @@
   "dec": 40,
   "ra_dec_error": 0.02,
   "image_energy_range": [0.5, 4.0],
-  "net_rate": 1.0,
-  "image_snr": 1.0
+  "image_snr": 1.0,
+  "net_count_rate": 1.0,
+  "backgound_count_rate": 0.2,
+  "rate_duration": 10,
+  "rate_energy_range": [0.5, 4.0]
```

Comment:
@modot, the net count rate would be for what duration? Energy range was mentioned in description earlier, it's fine if it's same as image energy range.

Reply:
The count rate serves as a reference for the flux of the source, while estimating the duration for WXT is challenging. The energy range is the same with that of the image.

Reply:
Ok, thank you. @jracusin shall we add additional comment for this count rate, will help users or merge as it is?

Reply:
@modot For users, would you like to add comment for net_count_rate using AdditionalInfo.schema from the core? This will prevent confusion with traditional calculation for GRB type events.

Reply:
@Vidushi-GitHub Sure, I'll create a new PR to add a comment.

Reply:
> @modot For users, would you like to add a comment for net_count_rate using AdditionalInfo.schema from the core? This will prevent confusion with traditional calculation for GRB type events.

@Vidushi-GitHub Would you please review the PR on https://github.com/nasa-gcn/gcn-schema/pull/139 ?